### DISCUSSION TITLE
Strip Led support

### DISCUSSION THREAD

**Discussion by barbatrukko** [sbidy/wiz_light#175] [General]
This integration can manage Starter kit strip LED 2 m?
I added to my house, I see in wiz app, but not in Home Assistant.

Also when I open documentation link from integration I receive "oh no! This page does not exist 😞"
https://www.home-assistant.io/integrations/wiz_light

**Comment by jcasarini** [comment]
Hi!
It won't autodiscover it, you have to add it manually in integrations.

![image](https://user-images.githubusercontent.com/6942398/135691605-a1647366-982e-4de2-98dd-ce65161eb3dd.png)

Have you tried that?

Cheers, 

Joaquin

**Reply by barbatrukko** [reply]
From where? I can't find "add" button in WIZ integration. How I can open this wizard?

**Reply by barbatrukko** [reply]
I found! I added strip by Wizard...work fine!

**Reply by jcasarini** [reply]
Great! Enjoy :)
